### DISCUSSION TITLE
Difference between subscription license and lifetime license and why should I bother with a subscription?

### DISCUSSION THREAD

**Discussion by ShaiAvr** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1325] [General]
I want to buy AlDente Pro, but the pricing model confuses me. You offer both an annual subscription and a single lifetime purchase which costs approximately the same as a subscription for 2 years. I don't understand why I should even bother buying a subscription instead of a lifetime license.
Will the lifetime license include future fees for major upgrades like Bartender or other apps, or is it truly a one-time purchase that gives me a license forever?
I couldn't find anything on the site that would indicate it's the former case and if it is the latter case, then why should I even bother buying a subscription instead of a lifetime purchase? The lifetime license seems to be better in the long term.

**Comment by MatthiasKerbl** [comment]
Hi @ShaiAvr,

Both license models include all future updates. If you want to use AlDente for the long term then it makes more sense in your case to buy the lifetime license.

Best,
Matthias
